app/src/hooks/useMarketMakerData.tsx

```diff
@@ -89,15 +91,17 @@ export const useMarketMakerData = (
       fee,
       isQuestionFinalized,
     ] = await Promise.all([
-      account
-        ? marketMaker.getBalanceInformation(account, outcomes.length)
+      account && cpk && cpk.address
```

Comment:
Is `account` necessary here?

Reply:
I think `yes` but not in that way. We need to  check for the account and if exist, we can instantiate the CPK, other way will throw an error. Also this is not working with the disconnected status. I fix this in one of the last commits